components/Home.module.css

```diff
@@ -48,6 +46,14 @@
     display: block;
 }
 
+@media screen and (min-width: 480px) {
+    .body {
+        width: 70%;
+        min-width: 360px;
+        max-width: 1000px;
```

Comment:
i think this part is making it not fit the entire screen, so i would take this out